### PR TITLE
SC-1817: Update command list to contain project specific items

### PR DESCRIPTION
**Pull request by alexanderM91**
- Ticket: https://spryker.atlassian.net/browse/SC-1817

### CODE REVIEW

File: generator/src/templates/deploy.bash.twig [hunk 1]
```diff
@@ -71,9 +71,6 @@ function showHelp()
     echo -e " ${GREEN}console${NC}\t  - Run Spryker console command, e.g. \`${ITALIC}${SPRYKER_SELF_SCRIPT} console code:test -vvv -g Acceptance\`${NC}."
```
Comment (sprymiker) [inline generator/src/templates/deploy.bash.twig]: Please change example to `console code:test -vvv -x Presentation`

File: generator/src/templates/deploy.bash.twig [hunk 2]
```diff
@@ -117,7 +122,7 @@ readonly USER_GID="1000"
 {% endif %}
 {% set isDevelopment = _mountMode != 'baked' %}
 {% if isDevelopment %}
-echo -e "*** ${WARN}DEVELOPMENT MODE${NC}"
+[ $1 != 'help' ] && echo -e "*** ${WARN}DEVELOPMENT MODE${NC}"
```
Comment (zyuzka) [inline generator/src/templates/deploy.bash.twig]: docker/deployment/default/deploy: line 124: [: !=: unary operator expected

Double brackets

File: generator/src/templates/deploy.bash.twig [hunk 3]
```diff
@@ -130,15 +135,15 @@ DOCKER_EXTRA_OPTIONS+=" -v ${SPRYKER_DOCKER_PREFIX}_${SPRYKER_DOCKER_TAG}_data_s
 
 function enableDebuggingMode()
 {
-    echo -e "*** ${WARN}DEBUGGING MODE${NC}"
+    [ $1 != 'help' ] && echo -e "*** ${WARN}DEBUGGING MODE${NC}"
```
Comment (zyuzka) [inline generator/src/templates/deploy.bash.twig]: same

File: generator/src/templates/deploy.bash.twig [hunk 4]
```diff
@@ -130,15 +135,15 @@ DOCKER_EXTRA_OPTIONS+=" -v ${SPRYKER_DOCKER_PREFIX}_${SPRYKER_DOCKER_TAG}_data_s
...
     export SPRYKER_XDEBUG_HOST_IP=$(which ip >/dev/null 2>&1 && ifconfig $(ip r | grep default | awk '{print $5}') | grep "inet " | awk '{print $2}' || echo 'host.docker.internal')
     DOCKER_COMPOSE_FILES+=" -f ${DEPLOYMENT_PATH}/docker-compose.xdebug.yml"
     DOCKER_EXTRA_OPTIONS+=" -v ${DEPLOYMENT_DIR}/context/php/conf.d/xdebug.ini:/usr/local/etc/php/conf.d/01-xdebug.ini:ro"
 }
 
 function enableTestingMode()
 {
-    echo -e "*** ${WARN}TESTING MODE${NC}"
+    [ $1 != 'help' ] && echo -e "*** ${WARN}TESTING MODE${NC}"
```
Comment (zyuzka) [inline generator/src/templates/deploy.bash.twig]: same

**Pull request merged**